src/components/layout/Navbar/vars.module.scss

```diff
@@ -0,0 +1,8 @@
+/* stylelint-disable property-no-unknown */
+@use 'src/styles/vars.scss' as vars;
+
+:export {
+  breakpointLg: vars.$breakpoint-lg;
+  breakpointMd: vars.$breakpoint-md;
+  breakpointSm: vars.$breakpoint-sm;
+}
```

Comment:
I really appreciate being thoughtful to avoid re-defining the same variable values but like we mentioned in the discussion on the Discord, I can see a few issues with this approach.

1. You re-exported all of the values specifically for this folder to use in the Navbar. If we need to use CSS variables in a different JS component and create a file like this again to access them in every single folder, I think that creates a messier codebase with redundant duplication than storing these variables at a global level somewhere in the `lib` folder.
2. Assuming we move these variables and import then from `src/lib` instead, I somewhat prefer writing the variables in JS than pulling from CSS because we only have to rewrite them a single time and we have the guarantee that their values exist. For example, you had a line elsewhere where you wrote `parseInt(vars.breakpointMd || '768')` and the OR variable assignment shouldn't be necessary because this is a constant rather than a dynamically set value.

A simple approach I can see working is just having

`src/lib/config.ts`

```
const config = {
  cssVars: {
    breakpointMd: 768
  }
}
```
This gives us the benefit of a second but still single source of truth for all CSS values we need to use in JS as long as we keep them synchronized, along with a guarantee that the variable exists as the exact integer we need, no additional files to generate, and we can easily reference the variable's source and value if we look at the code.